recipes/yasm/all/conanfile.py

```diff
@@ -52,6 +52,9 @@ def _build_vs(self):
                 msbuild.build_env.link_flags.append("/MACHINE:X86")
             elif self.settings.arch == "x86_64":
                 msbuild.build_env.link_flags.append("/SAFESEH:NO /MACHINE:X64")
+            if self.settings.build_type == "RelWithDebInfo":
+                self.output.info("YASM build type was {}, adjusting it to Release".format(self.settings.build_type))
+                self.settings.build_type = "Release"
```

Comment:
I think this can be moved to `configure` (or `config_options`) by doing:
```python
def configure(self):
    if self.settings.compiler == "Visual Studio" and self.settings.build_type == "RelWithDebInfo":
        self.output.info(f"yasm:build_type={self.settings.build_type} adjusted to Release")
        self.settings.build_type = "Release"
```
This will also make the package id of Release and RelWithDebInfo the same.

Reply:
@madebr I think that is also a good way to fix it. Actually just tried the same fix with libusb which has the same issue. Have you created an (updated) pull request for this?

Reply:
No, I haven't. Feel free to apply this change here.